.github/workflows/build-ios-prod.yml

```diff
@@ -0,0 +1,72 @@
+name: IOS Production Build
+
+on:
+  release:
+    types: [published]
```

Comment:
Can we replace this by a dispatch action for now? (until we test the CI for at least one manual release)
```yml
on: 
  workflow_dispatch:
```

Reply:
Sure thing